test/separate_compilation/jturner/use_array.chpl

```diff
@@ -1,4 +1,4 @@
-proc dothis(r:[1..2] real); 
+proc dothis(r:[1..2] real): real;
```

Comment:
Oh, I missed that this is a .future.  Would it be possible to add a .bad for it while you're here?